app/Services/Entity/EntityRelationService.php

```diff
@@ -233,7 +233,7 @@ protected function addRelations(Entity $entity): self
                 'attitude' => $relation->attitude,
                 'type' => 'entity-relation',
                 'is_mirrored' => $relation->isMirrored(),
-                'shape' => $relation->isMirrored() ? 'none' : 'triangle',
+                'shape' => $relation->isMirrored() && $relation->relation == $relation->mirror->relation ? 'none' : 'triangle',
```

Comment:
Missing a permission check if the mirrored relation is private from the user to view, causing a page crash `$relation->isMirrored() && $relation->mirror && $relation->relation == $relation->mirror->relation` :)